api/StaticRange.json

```diff
@@ -54,16 +54,16 @@
           "description": "<code>StaticRange()</code> constructor",
           "support": {
             "chrome": {
-              "version_added": "60"
+              "version_added": false
             },
             "chrome_android": {
-              "version_added": "60"
+              "version_added": false
             },
             "edge": {
-              "version_added": "18"
+              "version_added": null
             },
             "firefox": {
-              "version_added": false
+              "version_added": true
```

Comment:
#5104 sets this to "71" as per https://bugzilla.mozilla.org/show_bug.cgi?id=1575980, can we do the same?

Reply:
> "71"

OK, thanks — done